package homeroomassign;

public class LearnException {

	public static void main(String[] args) throws Exception {
		// TODO Auto-generated method stub

		int x=10;
		int y=0;
		int z;
		try {
			z = x/y;
		} catch (Exception e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
			
		
	}
System.out.println("exception");
	}}
